ui/components/policies/policies-view.js

```diff
@@ -14,17 +15,15 @@ export default function PoliciesView({ policies, count, topicsIds }) {
       <ThingCounterContainer count={count} singular={singular} plural={plural} />
       <ul className="policy-list list-reset">
         { policies.map(policy =>
-          <Policy key={policy.id} policy={policy} topicsIds={topicsIds} />,
+          <PolicyView key={policy.id} policy={new Policy(policy)} topicsIds={topicsIds} />,
```

Comment:
I'm rusty on my React, so I'm not sure if constructing a brand-new `Policy` every render is going to irritate React's innards and make it think that props are changing all the time, thus causing lots of extra pointless work. We should figure this out before merging, methinks.